src/pages/instances/forms/EditInstanceDetails.tsx

```diff
@@ -62,14 +61,19 @@ const EditInstanceDetails: FC<Props> = ({ formik, project }) => {
       {isClustered && (
         <Row>
           <Col size={12}>
-            <Input
+            <Select
               id="target"
               name="target"
-              type="text"
-              label="Instance location"
+              options={[
+                {
+                  label: formik.values.location,
+                  value: formik.values.location,
+                },
+              ]}
+              label="Location"
```

Comment:
I think it would be clearer to label this as "Cluster member"?

Reply:
Yes, updated it here and in a couple of other places to be consistent.